app/javascript/components/pages/TripEdit.js

```diff
@@ -65,7 +64,7 @@ export class TripEdit extends Component {
             <br />
             <FormGroup>
               <Label className="text-for-check">
-                Check here to show your trip to friends
+                Check to show your trip to friends
```

Comment:
```suggestion
                Check to show your trip to everyone
```
Just because using "friends" seems to suggest that the user can making connections to friends on the app when it doesn't support "friends" yet